index.bs

```diff
@@ -9432,15 +9428,12 @@ will not exist for the interface in the ECMAScript binding.
 The [{{NoInterfaceObject}}] extended attribute
 must [=takes no arguments|take no arguments=].
 
-If the [{{NoInterfaceObject}}] extended attribute
-is specified on an interface, then the [{{Constructor}}]
-extended attribute must not also be specified on that interface.
-A [{{NamedConstructor}}] extended attribute is fine,
-however.
-
 The [{{NoInterfaceObject}}] extended attribute
 must not be specified on an interface that has any
-[=static operations=] defined on it.
+[=constructors=] or [=static operations=] defined on it.
+
+Note: Combining the [{{NoInterfaceObject}}] and [{{NamedConstructor}}] extended attribute is not
+forbidden, however.
```

Comment:
Hmm, maybe it should be? I guess that would be a separate change.

Reply:
I wanna remove `[NoInterfaceObject]` anyway.